packages/office-ui-fabric-react/src/components/Keytip/examples/Keytips.CommandBar.Example.tsx

```diff
@@ -3,146 +3,103 @@ import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
-                    },
-                    keytipProps: keytipMap.SubmenuKeytip2,
-                    onClick: () => {
-                      console.log('test2');
-                    },
-                    subMenuProps: {
-                      items: [
-                        {
-                          key: 'testButton',
-                          text: 'Add to Outlook',
-                          keytipProps: keytipMap.SubmenuKeytip3,
-                          onClick: () => {
-                            console.log('test3');
-                          },
-                        },
-                        {
-                          key: 'testButton2',
-                          text: 'Go to Bing',
-                          keytipProps: keytipMap.SubmenuKeytip4,
-                          href: 'http://www.bing.com',
-                          target: '_blank',
-                        },
-                      ],
-                    },
-                  },
-                  {
-                    key: 'splitButtonTest',
-                    text: 'Other...',
-                    split: true,
-                    keytipProps: keytipMap.SubmenuKeytip5,
-                    subMenuProps: {
-                      items: [
-                        {
-                          key: 'splitButtonSubMenu1',
-                          text: 'Submenu Item w/o Keytip',
-                        },
-                        {
-                          key: 'splitButtonSubMenu2',
-                          text: 'Submenu Item w/o Keytip',
-                        },
-                      ],
-                    },
-                  },
-                ],
+              {
+                key: 'splitButtonSubMenu2',
+                text: 'Submenu Item w/o Keytip',
               },
-            },
-          ]}
-        />
-        {this.state.showMessageBar && (
-          <MessageBar messageBarType={MessageBarType.success}>Success Uploading</MessageBar>
-        )}
-        <Modal isOpen={this.state.showModal} onDismiss={this._hideModal} isBlocking={false}>
-          <h3>New Modal</h3>
-        </Modal>
-      </div>
-    );
-  }
-
-  private _showModal = (): void => {
-    this.setState({ showModal: true });
-  };
-
-  private _hideModal = (): void => {
-    this.setState({ showModal: false });
-  };
-
-  private _showMessageBar = (): void => {
-    this.setState({ showMessageBar: true });
-
-    // Hide the MessageBar after 2 seconds
-    setTimeout(this._hideMessageBar, 2000);
-  };
+            ],
+          },
+        },
+      ],
+    },
+  },
+];
 
-  private _hideMessageBar = (): void => {
-    this.setState({ showMessageBar: false });
-  };
-}
+export const KeytipsCommandBarExample: React.FunctionComponent = () => {
+  const [showModal, { toggle: toggleShowModal }] = useBoolean(false);
+  const [showMessageBar, { toggle: toggleShowMessageBar }] = useBoolean(false);
+  const commandBarItemProps = [
```

Comment:
This should probably be called `commandBarItems`, and in this case it would be worth doing `useMemo` because changing the identity of the command bar item array is likely to cause recalculations. (Once #12912 is merged you can do `useConst` instead. `useMemo` is needed right now because the `toggle` callback identity changes.)